portfolio/src/main/webapp/script.js

```diff
@@ -48,25 +48,24 @@ function bubbleFall() {
     }
 }
 
-// Get messages from the data servlet.
-function getMessages() {
-  fetch('/data').then(response => response.json()).then((messages) => {
-    console.log(messages);
+// Get comments from the data servlet.
+function getComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+    console.log(comments);
 
-    // Find container.
-    const messageContainer = document.getElementById('message-container');
-
-    // Add each message to container as list element.
-    for (let i = 0; i < 3; i++) {
-      messageContainer.appendChild(createListElement(messages[i]));
+    const commentSectionContainer = document.getElementById('comment-section-container');
+    
+    // Add each comment to container as comment element.
```

Comment:
This comment is redundant because this for loop is simple enough and the function/method names are specific enough that I can understand it without comments. 

Here's a snippet from an internal doc that you may or may not have access to (https://engdoc.corp.google.com/eng/doc/devguide/java/practices/comments.md?cl=head#what_for):

What comments are for
Use a comment when it is infeasible to make your code self-explanatory. Here's a great quote from Robert C. Martin's Clean Code:

The proper use of comments is to compensate for our failure to express ourself in code. Note that I used the word failure. I meant it. Comments are always failures. We must have them because we cannot always figure out how to express ourselves without them, but their use is not a cause for celebration.... Every time you express yourself in code, you should pat yourself on the back. Every time you write a comment, you should grimace and feel the failure of your ability of expression.

Self-explanatory code beats liberally-commented code (which, of course, beats code that can't be understood). If you need a comment to explain what a block of code does, try to introduce an explaining variable or to extract a method. If you've already done that but you still need a comment, try to rename it until it answers all your concerns. Sometimes, just choosing better identifier names can be enough; sometimes it takes more work. It's almost always worth it.